cli/tests/integration/inspector_tests.rs

```diff
@@ -410,22 +410,20 @@ async fn inspector_runtime_evaluate_does_not_crash() {
     ),
     WsRecv(r#"{"id":1,"result":{}}"#),
     WsRecv(r#"{"id":2,"result":{"debuggerId":"#),
-    WsSend(r#"{"id":3,"method":"Runtime.runIfWaitingForDebugger"}"#),
-    WsRecv(r#"{"id":3,"result":{}}"#),
```

Comment:
These two messages shouldn't be needed for this test to work (because we don't run with `--inspect-brk` flag, but with `--inspect` flag). However due to "handshaking" done in https://github.com/denoland/deno/blob/5027826a6407c96a973155be0620b458ab74d974/core/inspector.rs#L122-L125 it is currently required. This PR will attempt to fix that.